### ISSUE TITLE
Improvment: extend githubmoji for Gitlab

### ISSUE THREAD

**Issue by Lyokolux** [ma1ted/githubmoji#1]
Hey ! All is in the title. What do you think?

**Comment by malted** [comment]
This sounds like a good idea; might end up having to change the name :trollface:

**Comment by Lyokolux** [comment]
> This sounds like a good idea; might end up having to change the name :trollface:

Yeah ! I just through the idea as I find this extension very useful. I don't want to dedicate time to it right now. If someone is interested 😉
I think the core logic remains the same, but the CSS selectors, .... should be adjusted for Gitlab.

`Githublabmoji` → maybe too long
`Guthulamoji` → we loose the reference to github and gitlab

Or we can stick wit that name and it is good too :trollface:
